filebeat/module/postgresql/log/_meta/fields.yml

```
@@ -28,25 +43,75 @@
       description: >
         Name given to a query when using extended query protocol. If it is "<unnamed>", or not present,
         this field is ignored.
+    - name: command_tag
+      example: "SELECT"
+      description: >
+        Type of session's current command.
+        The complete list can be found at: src/include/tcop/cmdtaglist.h
+    - name: session_start_time
+      type: date
+      description: Time when this session started.
+    - name: virtual_transaction_id
+      description: Backend local transaction id .
+    - name: transaction_id
+      type: long
+      description: The id of current transaction.
+    - name: sql_state_code
+      # This code is not a number.
+      type: keyword
+      description: >
+        State code returned by Postgres (if any).
+        See also https://www.postgresql.org/docs/current/errcodes-appendix.html
+    - name: detail
+      description: >
+        More information about the message, parameters in case of a parametrized query.
+        e.g. 'Role \"user\" does not exist.', 'parameters: $1 = 42', etc.
+    - name: hint
+      description: >
+        A possible solution to solve an error.
+    - name: internal_query
+      description: internal query that led to the error (if any).
```

Please uppercase first letter of sentences in these descriptions.
```suggestion
      description: >
        Internal query that led to the error (if any).
```